shared/login/register/error/index.render.desktop.js

```diff
@@ -7,24 +7,52 @@ import {errorMap} from '../../../engine/errors'
 import openURL from '../../../util/open-url'
 
 const renderError = error => {
+  const fields = (error.raw && error.raw.fields || []).reduce((acc, f) => {
```

Comment:
we get this back in a kinda funky way, should likely move this into the action or smart component but i think we'll need a larger ticket to clean this up as we have a lot of this kind of logic in 'dumb' components